### PR TITLE
Add plugin code

### PR DESCRIPTION
**Pull request by ankit**
@turadg @eleith how does this look?

### CODE REVIEW

**Comment by eleith** [comment]
lovely. lgtm

**Comment by eleith** [comment]
do we have tests internally that could be ported?

**Comment by ankit** [comment]
No tests currently :| will add them over time.

File: src/TableUtils.js [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+  findCellPosition: function(table, targetTableCell) {
+    var body = table.childNodes[0];
+    var rows = Array.prototype.slice.call(body.childNodes);
+    var position = {
+      rowIndex: -1,
+      columnIndex: -1
+    };
+
+    rows.every(function(row, rowIndex) {
```
Comment (jnwng) [inline src/TableUtils.js:77]: [`every`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every) is intended to test every value in the array and return the evaluation of all of them, but you're not actually using the return of this invocation; are you looking for [`forEach`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach)? you can exit out of loops early in `forEach` as well if that was your intention.

Comment (jnwng) [inline src/TableUtils.js:77]: oh i see, you want to exit early from the loop the first time this returns false. got it.

File: src/scribe-plugin-table-command.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        var selection = new scribe.api.Selection();
+
+        var el = findBlockContainer(selection.range.endContainer);
+        var nextElement = el.nextSibling;
+
+        if (nextElement) {
+          scribe.el.insertBefore(tableElement, nextElement);
+        }
+
+        else {
```
Comment (jnwng) [inline src/scribe-plugin-table-command.js:35]: this is an odd "else", let's pull this up a couple lines to be consistent

**Comment by jnwng** [comment]
this doesn't block merge, but can we get a github page with a demo up so we can show this off?

**Comment by turadg** [comment]
LGTM.

+1 to a demo page, in a future revision. `gh-pages` is a bit of pain, in my experience. could it be made in a jsfiddle and linked from the README ?

**Comment by ankit** [comment]
great idea folks, i will add a demo page in another revision. Either `gh-pages` or `jsfiddle` sounds like a good idea, something that's playable with without checking out the repo.

File: src/scribe-plugin-table-command.js (hunk 2 above)
Comment (ankit) [inline src/scribe-plugin-table-command.js:35]: sounds good

**Comment by eleith** [comment]
yay. thanks for all the code @ankit =]

**Pull request merged**